test/support/adapter_assertions.ex

```diff
@@ -5,6 +5,24 @@ defmodule Cog.AdapterAssertions do
     assert response == expected_message
   end
 
+  # Because the raw template returns encoded json joined by newlines we have to
+  # add in newlines and then pull out the single result or return a list of
+  # result for those cases.
+  def assert_payload(%{"response" => response}, expected_payload) do
+    response = response
+    |> String.replace(~r/^}/m, "},")
+    |> String.rstrip(?,)
```

Comment:
I'd rather avoid using a positive lookahead, since I can't easily parse it at first glance. Although, the regex I wrote isn't much easier to understand. Maybe some more in-depth comments would be enough.